src/pytorch_lightning/loggers/mlflow.py

```diff
@@ -194,7 +204,7 @@ def run_id(self) -> str:
             The run id.
         """
         _ = self.experiment
-        return self._run_id
+        return self._run_id  # type: ignore[return-value]
```

Comment:
Can we not ignore types here? Since we are creating the experiment, we know `self._run_id` is not going to be `None` anymore so we can assert against it